### DISCUSSION TITLE
I didn't receive any offers for student pack, i have been approved

### DISCUSSION THREAD

**Discussion by m7mdful** [community/community#138253] [GitHub Education]
### Hi everyone,

I didn't receive any offers for student pack, i have been approved days ago.. is there a anyway to solve this ? if its a problem

**Comment by VadymMakohon** [comment]
Hey, m7mdful.
You are not alone, just be patient. I am sure you will get it resolved shortly.

**Reply by Emmanouil96** [reply]
same. I am approved but I can't access co pilot in VS studio. It directs me to payment page.

**Comment by davevad93** [comment]
Hi @m7mdful ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @m7mdful was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
